dask/dataframe/io/parquet/arrow.py

```diff
@@ -1547,11 +1569,14 @@ def _read_table(
 
     @classmethod
     def _arrow_table_to_pandas(
-        cls, arrow_table: pa.Table, categories, **kwargs
+        cls, arrow_table: pa.Table, categories, use_nullable_dtypes=False, **kwargs
     ) -> pd.DataFrame:
         _kwargs = kwargs.get("arrow_to_pandas", {})
         _kwargs.update({"use_threads": False, "ignore_metadata": False})
 
+        if use_nullable_dtypes:
+            _kwargs["types_mapper"] = PYARROW_NULLABLE_DTYPE_MAPPING.get
```

Comment:
More of an FYI if there is future appetite to get back a pandas DataFrame with _any_ pyarrow type, I think `to_pandas(..., type_mapper=...)` would go from arrow -> numpy -> arrow.

To avoid this conversion, I essentially split the `pa.Table` into `pa.ChunkedArray`s and stuck them into each column with as a `pd.ArrowExtensionArray`: https://github.com/pandas-dev/pandas/pull/49039/files#diff-868f7f48a0ed35429e240d9be0b98ad9303ceb2a7771b5bd21390eca332b0da4R267